README.md

```diff
@@ -4,7 +4,49 @@ Compare transformed TIMDEX records from two versions (A,B) of Transmogrifier.
 
 # abdiff
 
-TODO...
+`abdiff` is the name of the CLI application in this repository that performs an A/B test of Transmogrifier by 
```

Comment:
This is a good high-level overview, but I would leave as-is until we're done given the high likelihood of changes during the development process

Reply:
Good call.  That was my initial intention, but got away from me.  Removing steps 1-7 in favor of keeping it high level for now.